src/routers/OrganizationRouter.ts

```diff
@@ -122,6 +124,212 @@ const getAdminRouter = (basePath: string = '/', routePath: string = '/management
+        ScopesEnum.WriteAll,
+        ScopesEnum.WriteLocations,
+        ScopesEnum.WriteMetrics,
+        ScopesEnum.WriteCollections,
+        ScopesEnum.WriteLayers,
+        ScopesEnum.WriteWidgets,
+        ScopesEnum.WriteDashboards,
+        ScopesEnum.WriteUsers,
+        ScopesEnum.WriteOrganizations,
+      ];
+      const SCOPES_WRITE_DESCRIPTION =
+        'Modifying the resource in any way e.g. creating, editing, or deleting inside an organization.';
+
+      type Permission = { name: string; readScopes: string[]; writeScopes: string[]; description: string };
+
+      const PERMISSIONS: { [key: string]: Permission } = {
+        owner: {
+          name: 'Owner',
+          readScopes: [ScopesEnum.ReadAll],
+          writeScopes: [ScopesEnum.WriteAll],
+          description: 'Complete power over the assets managed by an organization.',
+        },
+        admin: {
+          name: 'Admin',
+          readScopes: [ScopesEnum.ReadAll],
+          writeScopes: [ScopesEnum.WriteAll],
+          description: 'Power over the assets managed by an organization.',
+        },
+        editor: {
+          name: 'Editor',
+          readScopes: [
+            ScopesEnum.ReadLocations,
+            ScopesEnum.ReadMetrics,
+            ScopesEnum.ReadCollections,
+            ScopesEnum.ReadLayers,
+            ScopesEnum.ReadWidgets,
+            ScopesEnum.ReadDashboards,
+          ],
+          writeScopes: [
+            ScopesEnum.WriteLocations,
+            ScopesEnum.WriteMetrics,
+            ScopesEnum.WriteCollections,
+            ScopesEnum.WriteLayers,
+            ScopesEnum.WriteWidgets,
+            ScopesEnum.WriteDashboards,
+          ],
+          description: 'Full content permission across the entire organization.',
+        },
+        viewer: {
+          name: 'Viewer',
+          readScopes: [
+            ScopesEnum.ReadLocations,
+            ScopesEnum.ReadMetrics,
+            ScopesEnum.ReadCollections,
+            ScopesEnum.ReadLayers,
+            ScopesEnum.ReadWidgets,
+            ScopesEnum.ReadDashboards,
+          ],
+          writeScopes: [],
+          description: 'Can view content managed by the organization',
+        },
+      };
+
+      const viewerGroupName = [groupName, 'VIEWER'].join('-');
+      const editorGroupName = [groupName, 'EDITOR'].join('-');
+      const adminGroupName = [groupName, 'ADMIN'].join('-');
+      const ownerGroupName = [groupName, 'OWNER'].join('-');
+
+      // create the main group + nested groups;
+      const root = await authzService.createGroup(groupName, groupDescription);
+      const owner = await authzService.createGroup(ownerGroupName, `${groupName} Owner`);
+      const admin = await authzService.createGroup(adminGroupName, `${groupName} Admin`);
+      const editor = await authzService.createGroup(editorGroupName, `${groupName} Editor`);
+      const viewer = await authzService.createGroup(viewerGroupName, `${groupName} Viewer`);
```

Comment:
We can optimize the calls to get them running in parallel:

- using `Promise.all` the resolve order is guaranteed when promises fulfil: 
```
const [root, owner, ...] = Promise.all([
    authzService.createGroup(groupName, groupDescription),
    authzService.createGroup(ownerGroupName, `${groupName} Owner`),
    ...
])